File: Unidad 5 Java/Programa6U5.java
package com.mycompany.programa6u5;
import java.util.Scanner;

public class Programa6U5 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese la longitud de la secuencia Fibonacci: ");
        int longitud = sc.nextInt();

        System.out.println("Secuencia Fibonacci:");
        for (int i = 0; i < longitud; i++) {
            System.out.print(fibonacci(i) + " ");
        }
    }

    static int fibonacci(int n) {
        if (n <= 1) {
            return n;
        } else {
            return fibonacci(n - 1) + fibonacci(n - 2);
        }
    }
}
        

File: Unidad 5 C#/Programa6U5.cs
using System;

namespace Programa6U5
{
    internal class Programa6U5
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese la longitud de la secuencia Fibonacci: ");
            int longitud = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Secuencia Fibonacci:");
            for (int i = 0; i < longitud; i++)
            {
                Console.Write(Fibonacci(i) + " ");
            }
        }

        static int Fibonacci(int n)
        {
            if (n <= 1)
            {
                return n;
            }
            else
            {
                return Fibonacci(n - 1) + Fibonacci(n - 2);
            }
        }
    }
}
        

